gears/src/types/tx/body.rs

```diff
@@ -73,7 +79,7 @@ impl<M: TxMessage> From<TxBody<M>> for inner::TxBody {
                 .map(|this| this.into())
                 .collect(),
             memo: tx_body.memo,
-            timeout_height: tx_body.timeout_height,
+            timeout_height: tx_body.timeout_height as u64, //TODO: consider using a copy of the raw TxBody struct which has a u32 timeout_height
```

Comment:
You may use it, but only if you need to change other fields too. It absolutely fine